### ISSUE TITLE
BitBucket: How to find out merge commit of pull request

### ISSUE THREAD

**Issue by SilverJan** [atlassian-api/atlassian-python-api#418]
Hi all,

I'm trying to get the merge commit (hash) of a merged pull request.

The BitBucket API 2 has such a field: https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Bworkspace%7D/%7Brepo_slug%7D/pullrequests/%7Bpull_request_id%7D

The BitBucket API 1 appareantly not: https://docs.atlassian.com/bitbucket-server/rest/5.16.0/bitbucket-rest.html

Anyone having a clue how this could be done?

**Comment by djachkov** [comment]
Hi, @SilverJan !
I didn't find a simple way, but you can use `get_commits(PROJ, REP, merges="only")` and then find specific commit parsing the "message" field:
```
'message': 'Merge pull request #34 in PROJ/repo '
             'from bugfix/az-azaza to master\n'
             '\n'
             "* commit '8dd1d6a47d15d5b016070c9abf65abae565c4ab0':\n"
             '  Touch azaza',
```
Will be available after #431 merging

**Comment by gonchik** [comment]
Hi! 
@djachkov  Thanks. 
I have upload a new release. 
Also, feel free to provide a PR for add some new things :)
